R/nneo_wrangle.R

```diff
@@ -0,0 +1,108 @@
+#############################################################################################
+#' @title Download and wrangle NEON's sensor-based data products
+
+#' @author Josh Roberti \email{jaroberti87@gmail.com}\cr
```

Comment:
@sckott Will the use of the \email{} above conflict with markdown use (comment below)?